target/linux/ramips/mt7621/base-files/lib/upgrade/ubnt.sh

```diff
@@ -8,32 +8,23 @@
 
 UBNT_ERX_KERNEL_INDEX_OFFSET=160
 
-ubnt_get_target_kernel() {
+ubnt_get_kernel_flag() {
 	local factory_mtd=$1
 	local current_kernel_index=$(hexdump -s $UBNT_ERX_KERNEL_INDEX_OFFSET -n 1 -e '/1 "%X "' ${factory_mtd})
```

Comment:
This tiny function can be merged into `ubnt_update_kernel_flag()`.

Reply:
sure